console/program/src/data/future/parse.rs

```diff
@@ -28,7 +28,7 @@ impl<N: Network> Parser for Future<N> {
             let (string, _) = Sanitizer::parse(string)?;
             // Parse the members.
             let (string, arguments) = separated_list0(
-                pair(pair(Sanitizer::parse, tag(",")), Sanitizer::parse),
+                pair(pair(Sanitizer::parse_whitespaces, tag(",")), Sanitizer::parse),
```

Comment:
Actually, I would push back on this. People sometimes will comment/document what each member represents.

Reply:
I don't disagree. In fact, I think we should allow comments and new lines everywhere. But right now the Aleo instructions parser enforces certain whitespace and comment restrictions by selectively using `Sanitizer::parse` (which skips over comments and whitespace including new lines) and `Sanitizer::parse_whitespace` (which skips over just whitespace excluding new lines). So this PR is making this part of the parser more consistent with other parts. Note that here one can still put comments just after the comma:
```
member, /* comment */
member, // comment
```
This PR just disallows
```
member /*comment*/ ,
```
or
```
member
,
```